mlflow/data/__init__.py

```diff
@@ -1,6 +1,8 @@
 import sys
 
+from . import sources as mlflow_data_sources
```

Comment:
Relative import required because the `mlflow.data` module isn't defined yet, so we can't import `mlflow.data.sources`.